internal/sdkprovider/service/project/organization_project.go

```diff
@@ -0,0 +1,276 @@
+package project
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	avngen "github.com/aiven/go-client-codegen"
+	"github.com/aiven/go-client-codegen/handler/organizationprojects"
+	retryGo "github.com/avast/retry-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/samber/lo"
+
+	"github.com/aiven/terraform-provider-aiven/internal/common"
+	"github.com/aiven/terraform-provider-aiven/internal/plugin/util"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig"
+)
+
+var ErrProjectStructureChangeNotSupported = fmt.Errorf("modifying parent_id, or billing_group_id is not supported - please use Aiven Console at https://console.aiven.io/ to perform this operation")
+
+var aivenOrganizationProjectSchema = map[string]*schema.Schema{
+	"organization_id": {
+		Type:        schema.TypeString,
+		Description: "ID of an organization. Changing this property forces recreation of the resource.",
+		Required:    true,
+		ForceNew:    true,
+	},
+	"project_id": {
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "Unique identifier for the project that also serves as the project name. Once set, cannot be changed.",
+	},
+	"billing_group_id": {
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "Billing group ID to assign to the project. It's required when moving projects between organizations.",
+	},
+	"parent_id": {
+		Type:     schema.TypeString,
+		Optional: true,
+		Computed: true,
+		Description: userconfig.Desc(
+			"Link a project to an [organization, organizational unit,](https://aiven.io/docs/platform/concepts/orgs-units-projects) or account by using its ID.",
+		).Referenced().Build(),
+	},
+	"technical_emails": {
+		Type:        schema.TypeSet,
+		Elem:        &schema.Schema{Type: schema.TypeString},
+		Optional:    true,
+		Description: "The email addresses for [project contacts](https://aiven.io/docs/platform/howto/technical-emails), who will receive important alerts and updates about this project and its services. You can also set email contacts at the service level. It's good practice to keep these up-to-date to be aware of any potential issues with your project.",
```

Comment:
nit: similar field in the project can as many as 10 items only, would clarify constraints. Also can be done after the release https://api.aiven.io/doc/#tag/Project/operation/ProjectCreate

Reply:
This is another endpoint. An at least in the description there is no restriction [here](https://api.aiven.io/doc/#tag/Organizations/operation/OrganizationProjectsUpdate).